### PR TITLE
Week 18 - project-mongo-api - Nadia Mazzoni

### PR DESCRIPTION
**Pull request by Bluepoisson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://simple-mongo-db.herokuapp.com/
+
+Endpoints:
+ https://simple-mongo-db.herokuapp.com/shows   /returns whole array of netflixdata 
+ https://simple-mongo-db.herokuapp.com/shows/81171862    /returns single show based on json.show-id 
+
+ https://simple-mongo-db.herokuapp.com/shows/director/546f6b61204d634261726f72   /returns show(s) by director db created objectId
```
Comment (luuf81) [inline README.md:20]: Clear and concise naming of endpoints 👍

Comment (luuf81) [inline README.md:20]: I liked the fact that you are using an endpoint with own generated db object id!

File: server.js [hunk 2]
```diff
@@ -3,36 +3,93 @@ import bodyParser from 'body-parser'
...
+const Director = mongoose.model('Director', {
+  name:String
+})
+
+const Show = mongoose.model('Show', {
+  show_id: Number,
+  title: String,
+  director: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'Director'
```
Comment (luuf81) [inline server.js:21]: Linking with other object, nice!

File: server.js [hunk 3]
```diff
@@ -3,36 +3,93 @@ import bodyParser from 'body-parser'
...
+app.get('/shows', async (req, res) =>  {
+  try {
+    const allShows = await Show.find();
+    if(allShows) {
+      res.json(allShows)
+    } else {
+      res.status(404).json({ error: 'Data not found' })
+    } 
+  } catch (err) {
+    res.status(400).json({ error: 'Invalid data' })
```
Comment (luuf81) [inline server.js:70]: nice error handling. I also like to use try/catch instead of .then..

**Review by luuf81 — APPROVED** [review]
Great job Nadia! 

Unfortunately I don't have many comments on room for improvements since it was a fairly small project compared to others and you have done very similar solutions as I. Like you mentioned in the readme, if you had more time you would have added more endpoints, intuitive search etc., I like that idea and might try it myself in future projects :)

You have clearly met the blue requirements in my opinion. Approved on my end!

//Olof

**Review by KarolCislowski — APPROVED** [review]
Nice Work 👍🏻

**Pull request closed without merging**